### PR TITLE
Feature: Make text area resisable in upload ontology and edit submission forms

### PR DESCRIPTION
**Pull request by Bilelkihal**
Related issue: 
https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/523

Capture d’écran 2024-05-31 à 10 26 39

### CODE REVIEW

**Review by syphax-bouazzouni — APPROVED** [review]

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
Need to e done here too 
image

**Review by Bilelkihal — COMMENTED** [review]
✅

File: app/helpers/submission_inputs_helper.rb [hunk 1]
```diff
@@ -444,7 +444,7 @@ def generate_list_text_input(attr, helper_text: nil, long_text: false)
 
     generate_list_field_input(attr, name, label, values, helper_text: helper_text) do |value, row_name, id|
       if long_text
-        text_area_tag(row_name, value, class: 'input-field-component', label: '')
+        text_area_tag(row_name, value, class: 'input-field-component', label: '', resize: true)
```
Comment (syphax-bouazzouni) [inline app/helpers/submission_inputs_helper.rb:447]: does this work ? 

`text_area_tag` is not our component `text_area_input`

Show a screenshot of the notes inputs bellow the description.

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

File: app/helpers/submission_inputs_helper.rb (hunk 1 above)
Comment (Bilelkihal) [reply app/helpers/submission_inputs_helper.rb]: No it doesn't work, I didn't notice that it's another component, done now:
Capture d’écran 2024-05-31 à 11 00 48

**Comment by syphax-bouazzouni** [comment]
@Bilelkihal what do you think of using this https://www.stimulus-components.com/docs/stimulus-textarea-autogrow ?

**Comment by Bilelkihal** [comment]
> @Bilelkihal what do you think of using this https://www.stimulus-components.com/docs/stimulus-textarea-autogrow ?

Great component, but I don't prefer using it here for the description & change notes, because users can fill it with a very long content which will make our form looks bad

**Pull request merged**